### PR TITLE
vmi details tab - upper page section (machine details)

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

> First half of details section, splitter for a couple of PR's so size of PR wont exceed 

## 🎥 Demo

image

### CODE REVIEW

File: src/views/virtualmachinesinstance/details/tabs/details/components/Details/CreateAt/CreateAt.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  const { t } = useKubevirtTranslation();
+  return (
+    <>
+      <DescriptionListTermHelpText>
+        <Popover
+          hasAutoWidth
+          maxWidth="30rem"
+          headerContent={t('Created At')}
+          bodyContent={
+            <Trans ns="'plugin__kubevirt-plugin'">
```
Comment (avivtur) [inline src/views/virtualmachinesinstance/details/tabs/details/components/Details/CreateAt/CreateAt.tsx:28]: is it on purpose with double and single quotes? maybe saving it as a const?

File: src/views/virtualmachinesinstance/details/tabs/details/components/Details/CreateAt/CreateAt.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                <BreadcrumbItem>metadata</BreadcrumbItem>
+                <BreadcrumbItem>creationTimestamp</BreadcrumbItem>
+              </Breadcrumb>
+            </Trans>
+          }
+        >
+          <DescriptionListTermHelpTextButton>{t('Created At')}</DescriptionListTermHelpTextButton>
+        </Popover>
+      </DescriptionListTermHelpText>
+      <DescriptionListDescription>created at</DescriptionListDescription>
```
Comment (avivtur) [inline src/views/virtualmachinesinstance/details/tabs/details/components/Details/CreateAt/CreateAt.tsx:50]: why not using Timestamp component?

Comment (metalice) [reply src/views/virtualmachinesinstance/details/tabs/details/components/Details/CreateAt/CreateAt.tsx]: waited for a merged, you right, i added

File: src/views/virtualmachinesinstance/details/tabs/details/components/Details/CreateAt/CreateAt.tsx (hunk 1 above)
Comment (metalice) [reply src/views/virtualmachinesinstance/details/tabs/details/components/Details/CreateAt/CreateAt.tsx]: fixed

**Pull request merged**